.github/workflows/test.yml

```diff
@@ -33,34 +32,53 @@ jobs:
           # giving us about 6 months of coverage.
           #
           # Minimum supported rust version (MSRV)
-          - "georust/geo-ci:rust-1.63"
+          - "ghcr.io/georust/geo-ci:rust-1.63"
           # Two most recent releases - we omit older ones for expedient CI
-          - "georust/geo-ci:proj-9.2.1-rust-1.71"
-          - "georust/geo-ci:proj-9.2.1-rust-1.72"
+          - "ghcr.io/georust/geo-ci:proj-9.6.2-rust-1.88"
+          - "ghcr.io/georust/geo-ci:proj-9.6.2-rust-1.89"
     container:
       image: ${{ matrix.container_image }}
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v4
-      - run: cargo install --version 1.6.0 cargo-all-features
-      - run: cargo build-all-features
-      - run: cargo test-all-features
-      - run: cargo build -p rstar-benches
+        uses: actions/checkout@v5
+      - name: Setup Rust cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          key: ${{ matrix.container_image }}
+      - run: cargo build --package rstar
+      - run: cargo test --package rstar
+
+  bench_and_demo:
+    name: build rstar benchmarks and demo
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v5
+      - name: Install stable toolchain
+        uses: dtolnay/rust-toolchain@stable
+      - name: Setup Rust cache
+        uses: Swatinem/rust-cache@v2
+      - name: Build bench
+        run: cargo build --workspace --package rstar-benches
```

Comment:
Are pure build tests really useful when we do Clippy builds for the whole workspace anyway? The main point here would be following through with codegen and linking?

Reply:
Removed the build job for `bench` and `demo` entirely since we do nothing with the result, as you say, and the Clippy job covers the build.